packages/cli/tests/unit/record/record.test.spec.ts

```diff
@@ -67,6 +79,64 @@ describe('record test', () => {
     });
   });
 
+  describe('record test with invalid directory parameter', () => {
+    it('shows error if directory parameter is file instead of directory', async () => {
```

Comment:
This test name is a little misleading. You're not actually testing if an error is shown. You're testing to make sure no error is thrown and that the `handler` function returns with `null`. Could you either rename the test or add logic for testing the output to the terminal?